systemtest/src/test/java/io/strimzi/systemtest/upgrade/AbstractUpgradeST.java

```diff
@@ -54,6 +56,7 @@
 public class AbstractUpgradeST extends AbstractST {
 
     private static final Logger LOGGER = LogManager.getLogger(AbstractUpgradeST.class);
+    private io.strimzi.systemtest.Constants systemtestConstants;
```

Comment:
Just a nit. `io.strimzi.systemtest.Constants` -> Constants

Reply:
There are already different Constants, so you need to do something like this - or you have to do `io.strimzi.systemtest.Constants.GLOBAL_TIMEOUT` etc. everywhere where it's needed.